camp/api/v1/monitors/tests.py

```diff
@@ -114,7 +114,8 @@ def test_create_entry(self):
         assert content['data']['fahrenheit'] is not None
 
         entry = Entry.objects.latest('timestamp')
-        assert entry.is_processed
+        assert entry.celcius = payload['AT(C)']
```

Comment:
I thought it was spelled `celsius`?

Reply:
Hahahahaha! 🤦‍♂️

Reply:
@stephengtuggy https://github.com/SJVAir/sjvair.com/pull/119